packages/trader/src/App/Containers/trade-header-extensions.tsx

```diff
@@ -1,31 +1,34 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { when } from 'mobx';
 import { MobileWrapper } from '@deriv/components';
 import { isMobile, routes, WS } from '@deriv/shared';
-import { connect, MobxContentProvider } from 'Stores/connect';
 import PopulateHeader from './populate-header';
+import { observer, useStore } from '@deriv/stores';
+import TraderProviders from '../../trader-providers';
+import { TCoreStores } from '@deriv/stores/types';
+
+type TradeHeaderExtensionsProps = {
+    store: TCoreStores;
+};
+
+const TradeHeaderExtensions = observer(({ store }: TradeHeaderExtensionsProps) => {
+    const { client, modules, ui, portfolio } = useStore();
```

Comment:
```suggestion
const TradeHeaderExtensions = observer(() => {
    const stores = useStore();
    const { client, modules, ui, portfolio } = stores;
```
@jim-deriv You can access the store via the `useStore` hook.

Reply:
@farzin-deriv the thing is TradeHeaderExtensions accepts root_store as a `store` prop and passes it down to its child, i suggest that we keep this functionality as is🙂

Reply:
As I mentioned we can get `root_store` from the `useStore` hook and there is no need to pass `props` to the component 🙇🏻

Reply:
@farzin-deriv, naturally, I'd choose to preserve the structure of the `component` as well, but I'll leave this decision to you and @maryia-deriv. Whatever you guys decide to go with, I'll implement. However, one question I'd ask myself is that  why didn't the original authors use `connect` directly on those components, but rather decide to pass the `store` `object` as a `prop` to the `component`. If I can't come up with an obvious answer to that, then I won't implement the change  until i'm quite sure of the reason.

Reply:
@jim-deriv Let's keep it as it is as this PR is already big and many other cards are waiting for these changes, After this one is merged we can investigate more on this separately 🙇🏻